apisix/core/utils.lua

```diff
@@ -337,6 +337,24 @@ do
     end
 end
 -- Resolve ngx.var in the given string
+-- @function core.utils.resolve_var
```

Comment:
Are these comments necessary?

Reply:
I think we can keep it, but we need to make sure it's correct.